cmd/agent/app/options/options.go

```diff
@@ -181,6 +183,8 @@ func (o *Options) AddFlags(fs *pflag.FlagSet, allControllers []string) {
 	fs.StringVar(&o.KarmadaKubeConfig, "karmada-kubeconfig", o.KarmadaKubeConfig, "Path to karmada control plane kubeconfig file.")
 	fs.StringVar(&o.KarmadaContext, "karmada-context", "", "Name of the cluster context in karmada control plane kubeconfig file.")
 	fs.StringVar(&o.ClusterName, "cluster-name", o.ClusterName, "Name of member cluster that the agent serves for.")
+	fs.StringArrayVar(&o.ClusterLables, "cluster-labels", []string{}, "Lables of the member cluster")
+	fs.StringArrayVar(&o.ClusterAnnotations, "cluster-annotations", []string{}, "Annotations of the member cluster")
```

Comment:
It would be great to have a more detailed description, because it's not clear how to specify multiple key-value pairs.

Reply:
Thanks. I updated to https://github.com/karmada-io/karmada/pull/6172/files#diff-06009d1be092147fe694efde678466f0c2f7771ba92e63ca27b912fe8e846606R186-R187. PTAL